src/components/PrivacyPolicy/PrivacyPolicyContent.js

```diff
@@ -0,0 +1,397 @@
+/* eslint-disable max-len */
+import React from "react";
+
+const PrivacyPolicyContent = () => (
+    <div id="doc" style={{ fontFamily: "Poppins, Roboto, sans-serif" }}>
```

Comment:
Same for style removal. Use `<Typography>` https://v4.mui.com/pt/components/typography/